frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx

```diff
@@ -47,17 +50,28 @@ const DashboardInfoSidebar = ({
     PLUGIN_CACHING.isEnabled() && MetabaseSettings.get("enable-query-caching");
 
   const handleDescriptionChange = useCallback(
-    async (description: string) => {
-      await setDashboardAttribute("description", description);
+    (description: string) => {
+      setDashboardAttribute("description", description);
```

Comment:
this isn't an async function so we don't need to `await` it
https://github.com/metabase/metabase/blob/3fb962b9b816112a33f94079b63abe862dc7fe3f/frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx#L192-L197

Reply:
I think it waits for action to be dispatched and proceed by the redcuers? Not sure if this change affects something or not

Reply:
I'm pretty sure it doesn't wait for anything asynchronous. The referenced function isn't `async` so it doesn't matter what's even defined inside the function.

This in turn calls https://github.com/metabase/metabase/blob/734c39090455f64826e824ab59e51a76b999f649/frontend/src/metabase/dashboard/actions/core.js#L14

which is a sync action anyway. So we're getting the same result even if https://github.com/metabase/metabase/blob/3fb962b9b816112a33f94079b63abe862dc7fe3f/frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx#L192-L197 is to be an `async` function.

i.e. without an `await` this should already wait for the reducer to update all the values since it's a sync action.

my reason for the change is to avoid confusion. Async actions kind of imply some API calls would happen down the line. But there's none in this case.